empty.yaml

```diff
@@ -0,0 +1,4 @@
+# This is an empty yaml file so make lint-yaml passes.
+# lint-yaml removes files with {{ in them using grep -L.
```

Comment:
I still don't understand why we can't fix this properly. Change the script to something like this:
```
find . -name "*.yml" -not -exec grep "{{" {} \; -print
```

Reply:
I've tried a version of Frank's suggestion in the common-files as a test. We wouldn't actually check that in here, only this repo's Makefile change once the common-files change is propagated.